src/app/manager-dashboard/manager-dashboard.component.html

```diff
@@ -20,7 +20,9 @@
     <ng-container *ngIf="requestStatus === 'accepted'">
       <a routerLink="sync" i18n mat-raised-button>Manage Sync</a>
       <a routerLink="/upgrade" *ngIf="versionLocal !== versionParent" i18n mat-raised-button>Upgrade</a>
+      <a routerLink="/upgrade/myplanet" *ngIf="versionLocal === versionParent" i18n mat-raised-button>Upgrade myPlanet</a>
     </ng-container>
+    <a routerLink="/upgrade/myplanet" *ngIf="" i18n mat-raised-button>Upgrade myPlanet</a>
```

Comment:
This is never shown because of the `*ngIf`, and would be a duplicate of the other button on line 23 anyway.  I think this line can be removed

Reply:
yes it was not supposed to be there